### PR TITLE
Add Mothroach Burger

### PR DESCRIPTION
**Pull request by TurboTrackerss14**
## About the PR
Adds Mothroach Burger to the game.

## Why / Balance
Threatened discord users with this PR if they didn't stop spamming the mothroach gifs.

## Technical details
Whats technical about a delicious burger?

## Media
https://github.com/space-wizards/space-station-14/assets/130304754/bed8a560-4cd5-492a-8245-6b1fe4e18e08
- [x ] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added Mothroach Burger.

### CODE REVIEW

**Comment by UbaserB** [comment]
honestly this is funny

**Comment by MrGloopy** [comment]
DO NOT THE BABY NOOOOOOOO

**Comment by Hanzdegloker** [comment]
This brings me great sadness

**Comment by LankLTE** [comment]
it it possible to make it do the moth scream when you eat it

File: Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml [hunk 1]
```diff
@@ -891,3 +891,31 @@
...
+        reagents:
+        - ReagentId: Nutriment
+          Quantity: 3
+        - ReagentId: Protein
+          Quantity: 4
+        - ReagentId: Vitamin
+          Quantity: 7
+  - type: Tag
+    tags:
+    - Meat
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml:921]: No line ending is triggering my little brain

File: Resources/Prototypes/Flavors/flavors.yml [hunk 2]
```diff
@@ -883,3 +883,8 @@
   id: horrible
   flavorType: Base
   description: flavor-base-horrible
+
+- type: flavor
+  id: light
+  flavorType: Complex
+  description: flavor-complex-light
```
Comment (VasilisThePikachu) [inline Resources/Prototypes/Flavors/flavors.yml:890]: Ditto

File: Resources/Textures/Objects/Consumable/Food/burger.rsi/meta.json [hunk 3]
```diff
@@ -1,7 +1,7 @@
 {
   "version": 1,
   "license": "CC-BY-SA-3.0",
-  "copyright": "Taken from tgstation and modified by Swept and potato1234x at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, ian.png created by EmoGarbage",
+  "copyright": "Taken from tgstation and modified by Swept and potato1234x at https://github.com/tgstation/tgstation/commit/40d75cc340c63582fb66ce15bf75a36115f6bdaa, ian.png created by EmoGarbage, mothroach.png created by TurboTracker",
```
Comment (VasilisThePikachu) [inline Resources/Textures/Objects/Consumable/Food/burger.rsi/meta.json:4]: Did you press enter here or did github do a funny line wrap

**Review by VasilisThePikachu — CHANGES_REQUESTED** [review]

File: Resources/Textures/Objects/Consumable/Food/burger.rsi/meta.json (hunk 3 above)
Comment (VasilisThePikachu) [reply Resources/Textures/Objects/Consumable/Food/burger.rsi/meta.json:4]: Ok github did a funny line wrapping ignore this

File: Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml (hunk 1 above)
Comment (TurboTrackerss14) [reply Resources/Prototypes/Entities/Objects/Consumable/Food/burger.yml:921]: good

**Comment by TheShuEd** [comment]
https://github.com/space-wizards/space-station-14/issues/8524

check it out.  We no longer accept microwave recipes.

**Comment by lzk228** [comment]
> #8524
> 
> check it out. We no longer accept microwave recipes.

i remember someone said like
> we can add but you should now it can be deleted at any time for refactor

**Comment by VasilisThePikachu** [comment]
> #8524
> 
> check it out. We no longer accept microwave recipes.

This you?
#22548 #21639

**Comment by LankLTE** [comment]
To my understanding the point of the microwave recipe freeze isn't "you can't add these", it's more just "when chef rework happens these may end up getting deleted." So, yeah, just keep that in mind and we should be all good.

**Comment by VasilisThePikachu** [comment]
> :trollface: fixed whitespace @VasilisThePikachu

https://youtu.be/g44Pq1zOlt0?si=E_InBS9cFoggTgJM&t=19

**Review by VasilisThePikachu — APPROVED** [review]
You outsmarted me

**Review by nikthechampiongr — APPROVED** [review]
Looks fine and it also means more suffering for those who buy a morbillion mothroach crates.

**Review by lzk228 — APPROVED** [review]
looks fine to me

**Review by deltanedas — APPROVED** [review]
we are doing sybil attack

**Comment by nikthechampiongr** [comment]
By the way, don't you need to add this to the guidebook entry?

**Comment by LankLTE** [comment]
Cooking guidebook isn't supposed to have every recipe iirc, just some basic examples.

**Comment by TurboTrackerss14** [comment]
Nik and Vasilis bullying me into code diving for guidebook entries again just for it to not even be a full guidebook entry. 
If it gets merged someone remind me to make a wiki update or pester the wikigoblins to do it for me

**Pull request merged**